Sources/Core/Sources/Render/Mesh/ChunkSectionMesh.swift

```diff
@@ -24,24 +24,25 @@ public struct ChunkSectionMesh {
     transparentAndOpaqueMesh.clearGeometry()
     translucentMesh.clear()
   }
-  
+    
   /// Encode the render commands for this chunk section.
   /// - Parameters:
   ///   - position: Position the mesh is viewed from. Used for sorting.
-  ///   - sortTranslucent: Whether the translucent mesh should be sorted or not.
-  ///   - transparentAndOpaqueEncoder: Encoder for rendering transparent and opaque geometry.
-  ///   - translucentEncoder: Encoder for rendering translucent geometry.
+  ///   - renderTranslucent: Defines whether translucent or transparent and opaque mesh is rendered.
+  ///   - worldRenderEncoder: Encoder for rendering geometry.
   ///   - device: The device to use.
   ///   - commandQueue: The command queue to use for creating buffers.
   public mutating func render(
```

Comment:
I think it would be best to have this as two separate functions, because this way ends up having a bunch of unnecessary arguments when reddening the transparent and opaque mesh.  And the renderTranslucent sounds like it's whether to render translucent geometry as well as opaque and transparent geometry. And while you're at it just render worldRenderEncoder to just renderEncoder, it doesn't necessarily have to be the world render encoder.